Sources/SwiftSMTP/SMTP.swift

```diff
@@ -99,13 +99,13 @@ public struct SMTP {
     ///     - mail: `Mail` object to send.
     ///     - completion: Callback when sending finishes. `Error` is nil on success. (optional)
     public func send(_ mail: Mail, completion: ((Error?) -> Void)? = nil) {
-        send([mail]) { (_, failed) in
+        send([mail], completion:  { (_, failed) in
```

Comment:
Just curious, were these syntax changes necessary?

Reply:
Necessary, no - it was just XCode 12.2 (12B45b) complaining with a warning: 
> Backward matching of the unlabeled trailing closure is deprecated; label the argument with 'completion' to suppress this warning

Reply:
Oh interesting.  I'll make a note to review that [Issue 116](https://github.com/Kitura/Swift-SMTP/issues/116)